Add recap blog post for SQL training 1 (#553)
trinodb/trino.io: _posts/2023-10-18-sql-training-1.md

---
layout: post
title: "Getting started with Trino and SQL"
author: "Manfred Moser"
excerpt_separator: <!--more-->
image: /assets/trino-sql.png
show_pagenav: false
---

In our training series [Learning SQL with Trino from the experts]({% post_url
2023-09-27-training-series %}) Martin Traverso, Dain Sundstrom, David Phillips,

> and myself run through the wide range of SQL support and features of Trino with

Reviewer: ```suggestion
and myself will run through the wide range of SQL support and features of Trino with
```
Hasn't all happened yet! Let's make it clear it's still mostly upcoming.

Author: Done

> our audience. In the first episode we covered the concepts of Trino and SQL, and

Reviewer: ```suggestion
our audience. In the first episode, we covered the concepts of Trino and SQL, and
```

Author: Done

then started to learn some basic SQL. Now you can take advantage of the
recording and available resources to learn at your own pace.

<!--more-->

The recording of the event allows you to watch it all as if you attended live,

> jump to specific section as desired, or pause while you follow along with the

Reviewer: ```suggestion
jump to specific sections as desired, or pause while you follow along with the
```

Author: Done

demos:

{% youtube SnvSBYhRZLg %}

Following are a couple of specific timestamps for interesting
topics snippets:

* [What is Trino?](https://www.youtube.com/watch?v=SnvSBYhRZLg&t=380)
* [Catalogs and connectors](https://www.youtube.com/watch?v=SnvSBYhRZLg&t=1163)
* [Clients](https://www.youtube.com/watch?v=SnvSBYhRZLg&t=1658)
* [SQL WHERE statement](https://www.youtube.com/watch?v=SnvSBYhRZLg&t=3224)

> Find more on the description on YouTube.

Reviewer: ```suggestion
The full timestamps for every part of the talk are in the description on
YouTube, making it easy to skip to the sections that interest you.
```

Author: Done

Also make sure you take advantage of these additional resources:

* [General overview slide deck for the series]({{site.baserurl}}/assets/blog/sql-training-series-starburst-2023.pdf), with links to resources like our [community chat]({{site.baserurl}}/slack.html)
* Slides deck for [SQL and Trino concepts](https://trinodb.github.io/presentations/presentations/sql-trino/index.html)
* Slide deck for [SQL basics with Trino](https://trinodb.github.io/presentations/presentations/sql-basics/index.html), including a file with all SQL statements ready to go
* [Trino: The Definitive Guide]({{site.baserurl}}/trino-the-definitive-guide.html)

> Now that you know of the series and saw the first iteration, make sure you

Reviewer: ```suggestion
Now that you know of the series and saw the first part of it, make sure you
```

Author: Done
